### ISSUE TITLE
Specify contributing guidelines?

### ISSUE THREAD

**Issue by pedrohba1** [translucent-link/blackbird#2]
I've not seen many external adapters in golang and I would like to contribute with many examples.

Maybe you should specify some guidelines on it.

**Comment by ijonas** [comment]
That's very kind of you and I'll draft some CONTRIB guidelines in the next couple of days.

My initial thoughts were to keep this repo focused on being a template. Rather than being a container for a collection of go-based adapters. 

I'm not a big fan of a large example external adapters as is done in the Chainlink repo: https://github.com/smartcontractkit/external-adapters-js/tree/develop/packages/sources

I'd rather keep this repo small and focused and the dependencies to a minimum. 

However I do want to showcase other Go-based EAs by creating a section in the README.md that provides links and a description to other repos.

We do however welcome contributions. There's plenty of work to be done on the template, not least caching responses and rate limiting functionality.

I would love to know your thoughts.

**Comment by pedrohba1** [comment]
>However I do want to showcase other Go-based EAs by creating a section in the README.md that provides links and a description to other repos.

Ok. Then I can make some examples and create a pull request to insert them into the template `README.md`, right?

**Comment by ijonas** [comment]
Yes, absolutely. I've taken the liberty to kick off the list of Go-based adapters at the bottom of the `README.md`.
